### PR TITLE
Add Avalonia. Remove deprectated ..container

### PR DESCRIPTION
**Pull request by MongoCaleb**
### Jira

https://jira.mongodb.org/browse/DOCSP-30965

### Staged Changes

https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/docsp-30965/sdk/dotnet/
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/docsp-30965/sdk/dotnet/model-data/data-binding/
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/docsp-30965/sdk/dotnet/install/

### CODE REVIEW

File: source/sdk/dotnet.txt [hunk 1]
```diff
@@ -85,119 +85,113 @@ Develop Apps with Realm
    .. tab:: Use Realm Locally
       :tabid: local-realm
 
-      .. container::
+      Use free open-source Realm as a local object store on a device.
```
Comment (papafe) [inline source/sdk/dotnet.txt:88]: I am not sure of what is changing with the container, but looking at the comparison between the staged build and the one currently online I can see that in the staged one the phrase "Use free open-source Realm as a local object store on a device." is right aligned, while the rest is left aligned.

File: source/sdk/dotnet/install.txt [hunk 2]
```diff
@@ -10,14 +10,14 @@ Install Realm - .NET SDK
    :depth: 2
    :class: singlecol
 
-Realm's .NET SDK enables development of applications in C# for the .NET, 
+You can use Realm's .NET SDK to develop apps in C# for many platforms:
```
Comment (papafe) [inline source/sdk/dotnet/install.txt:13]: Same comment as on the top

File: source/sdk/dotnet.txt [hunk 3]
```diff
@@ -31,11 +31,11 @@ Realm .NET SDK
    Release Notes <https://github.com/realm/realm-dotnet/releases>
 
 
-Use the Realm .NET SDK to develop client applications written in C# for .NET,
+You can use Realm's .NET SDK to develop apps in C# for many platforms:
```
[the remark this answers is not in the document]
Comment (MongoCaleb) [reply source/sdk/dotnet.txt:34]: They're all frameworks, so I used that.

File: source/sdk/dotnet/model-data/data-binding.txt [hunk 4]
```diff
@@ -10,8 +10,8 @@ Data Binding - .NET SDK
    :depth: 2
    :class: singlecol
 
-In Xamarin and MAUI applications, data binding enables UI updates when an 
-underlying object or collection changes, and vice versa.
+In .NET applications (Xamarin, MAUI, Avalonia UI, and others) data binding enables
```
[the remark this answers is not in the document]
Comment (MongoCaleb) [reply source/sdk/dotnet/model-data/data-binding.txt:13]: much better!

**Review by papafe — APPROVED** [review]
Looks good!

File: source/sdk/dotnet/model-data/data-binding.txt [hunk 5]
```diff
@@ -10,8 +10,9 @@ Data Binding - .NET SDK
    :depth: 2
    :class: singlecol
 
-In Xamarin and MAUI applications, data binding enables UI updates when an 
-underlying object or collection changes, and vice versa.
+In .NET applications built with Xamarin, MAUI, Avalonia UI and other MVVM-based 
+frameworks, data binding enables UI updates when an underlying object or 
+collection changes, and vice versa.
```
Comment (cbush) [inline source/sdk/dotnet/model-data/data-binding.txt:15]:
```suggestion
collection changes and vice versa.
```

**Review by cbush — APPROVED** [review]

**Pull request merged**